### ISSUE TITLE
Redis v6.0.10 aarch64 issue

### ISSUE THREAD

**Issue by langrenfengzi** [termux/termux-packages#6581]
Run the command **redis-server** got the following errors:
```
=== REDIS BUG REPORT START: Cut & paste starting from here ===
21169:M 31 Mar 2021 12:42:52.275 # === ASSERTION FAILED ===
21169:M 31 Mar 2021 12:42:52.275 # ==> server.c:4866 '!ret' is not true
```
Termux v0.108 and redis v6.0.10
```
$apt list redis
Listing... Done
redis/stable,now 6.0.10 aarch64 [installed]
```
It seems to have a bug in this version.
See [redis pod does not work on arm64 kube worker node](https://github.com/redis/redis/issues/8351)

In the long past it was ok. I guess the redis was upgrade to v6 from v5 after I run the command **pkg upgrade**.
Could you update the redis package to the latest v6.2.x? It maybe fixed on this.
Or could you tell me how to get the v5.x redis binary?

**Comment by Grimler91** [comment]
The assert was introduced in https://github.com/redis/redis/commit/8ff022636176e85e2ced75223fe64f95701772e8#diff-1abc5651133d108c0c420d9411925373c711133e7748d9e4f4c97d5fb543fdd9. I don't know what the proper fix would be here (reverting that commit probably works, but it was applied for a reason)

**Comment by langrenfengzi** [comment]
Ok. Thanks for your reply. I'll still wait for your fix.

**Comment by Grimler91** [comment]
@langrenfengzi @Fernal73, Issue seem to be fixed in redis 6.2.2, please try it when [the build](https://github.com/termux/termux-packages/actions/runs/802399130) finishes and let me know if you still experience problems.
